src/COCOA/wwwroot/js/components/CocoaHeader.jsx

```diff
@@ -10,7 +10,7 @@ class CocoaHeader extends React.Component {
         super(props);
     }
     render() {
-        return (<Navbar inverse collapseOnSelect fluid>
+        return (<Navbar inverse fluid>
```

Comment:
Ref. to conversations. Not using collapseOnSelect because of [React-Bootstrap href-bug](https://github.com/react-bootstrap/react-bootstrap/issues/2470).